clients/cobol-lsp-vscode-extension/src/services/Middleware.ts

```diff
@@ -0,0 +1,32 @@
+import {CancellationToken} from "vscode";
```

Comment:
header is missing - guys double check if you have configured the IDE to generate header every time you create a new file :)

Reply:
Yes, I've configured it in IDEA, but not in VSCode. I'll fix it.